ee/modules/110-istio/templates/control-plane/iop.yaml

```diff
@@ -1,3 +1,10 @@
+{{ $supportedProviders := list "aws" "gcp" "azure" }}
+{{ $currentProvider := .Values.global.clusterConfiguration.cloud.provider | lower }}
```

Comment:
It will fail in static cluster. There isn't a `cloud` section.

Reply:
i didn't know about it, ok i wil fix it